tests/phpunit/sitemaps-renderer.php

```diff
@@ -50,19 +50,19 @@ public function test_get_sitemap_index_stylesheet_url_pretty_permalinks() {
 	public function test_get_sitemap_index_xml() {
 		$entries = array(
 			array(
-				'loc'     => 'http://' . WP_TESTS_DOMAIN . '/wp-sitemap-posts-post-1.xml',
```

Comment:
These changes in the unit test should also be reverted (including those for the `$expected` value below).  They pass as is, but  think it would be better for long term maintenance of the unit tests if the convention used in this test were what was used for the actual sitemap index.

Reply:
Agreed!